build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks.csproj

```diff
@@ -8,6 +8,7 @@
     <RootNamespace>Xamarin.Android.Tools.BootstrapTasks</RootNamespace>
     <AssemblyName>Xamarin.Android.Tools.BootstrapTasks</AssemblyName>
     <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion Condition="'$(MSBuildAssemblyVersion)' == '16.0'">4.7.2</TargetFrameworkVersion>
```

Comment:
Why make this conditional instead of just updating the previous line?

For that matter, if *all* of these projects potentially need to be updated to appease VS, would it work if these instead came from `Configuration.props`?  For example, update `Configuration.props` to contain:

```xml
<DefaultDesktopFrameworkVersion>v4.7.2</DefaultDesktopFrameworkVersion>
```

Then within e.g. `Xamarin.Android.Tools.BootstrapTasks.csproj`:

```xml
<Import Project="..\..\Configuration.props" />
<PropertyGroup>
  <TargetFrameworkVersion>$(DefaultDesktopFrameworkVersion)</TargetFrameworkVersion>
</PropertyGroup>
```

I don't know if VS-the-IDE would like this, but `msbuild` probably would be fine with it.

Reply:
Shouldn't this (and other!) lines have a `v` prefix, e.g. `v4.7.2`, not just `4.7.2`?